src/defines.asm

```diff
@@ -20,9 +23,6 @@
 !ram_vertical_speed = $7FFB3C
 !ram_mb_hp = $7FFB3E
 !ram_enemy_hp = $7FFB40
-!ram_shine_counter_1 = $7FFB30 ; goes 1-A
-!ram_shine_counter_2 = $7FFB14 ; armed shine duration
-!ram_shine_counter_3 = $7FFB1A ; armed shine duration 2
```

Comment:
Let me know if want more explanation of changes.

Here I got rid of shine_counter_1 because it goes 1-A like the comment said... and that's it... I can't find any other reason for it to exist.  I gave the other two variables better names.

Also one of the variables (shine_dash_held_late) is now used for shine tune feature.